src/segment_media_others.go

```diff
@@ -0,0 +1,7 @@
+// +build !windows
+
+package main
+
+func (s *media) enabled() bool {
```

Comment:
Rather than doing this I would for now only enable the segment on Windows by validating the platform we're on. If we ever have a fully working set on another platform, we can always extend it like this.